app/src/main/java/com/fastlink/zekrapp/appData/data/Categories.kt

```diff
@@ -9,8 +10,8 @@ fun getCateories(){
         categoriesData.add(
             CategoryModel(
```

Comment:
I already mentioned the name of the class, category is not clear, category of what?

Reply:
Sorry About that i will change it soon